druid/src/core.rs

```diff
@@ -359,7 +367,7 @@ impl<T, W: Widget<T>> WidgetPod<T, W> {
     ///
     /// Returns `true` if the hot state changed.
     ///
-    /// The provided `child_state` should be merged up if this returns `true`.
+    /// The WidgetPod should merge up its state when `true` is returned.
     fn set_hot_state(
```

Comment:
This method's comment is still stale. The very first sentence talks about `rect` which is no longer a parameter.